webrtc-stats.html

```diff
@@ -1293,6 +1294,26 @@ <h2>
                 </p>
                 
               </dd>
+              <dt>
+                <dfn>estimatedMinimumJitterBufferTargetDelay</dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  There are various reasons why the jitter buffer delay might be increased to a higher value, such as
+                  to achieve AV synchronization or because a
+                  <a href="https://w3c.github.io/webrtc-extensions/#dom-rtcrtpreceiver-playoutdelay">playoutDelay</a>
+                  was set on a RTCRtpReceiver. When using one of these mechanisms, it can be useful to keep track of
+                  the minimal jitter buffer delay that could have been achieved, so WebRTC clients can track the amount
+                  of additional delay that is being added.
+                </p><p>
+                  This metric works the same way as {{jitterBufferTargetDelay}}, except that it is not affected by
+                  external mechanisms that increase the jitter buffer target delay, such as playoutDelay (see link above),
+                  AV sync, or any other mechanisms. This metric is purely based on the network characteristics such
+                  as jitter and packet loss, and can be seen as the minimum obtainable jitter buffer delay if no
+                  external factors would affect it.
+                </p>
+              </dd>
```

Comment:
Could we explicitly state that the metric is updated every time the `jitterBufferEmittedCount` is updated. This is the jitter buffer delay if there was no reason for holding the sample or frame for application related reasons.

Reply:
Would CPU issues cause this metric to be updated or not?